tests/text/test_bertscore.py

```diff
@@ -1,12 +1,21 @@
-from typing import Any
+import os
+from typing import Any, Dict, List
 
 import numpy as np
 import pytest
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
 
-from torchmetrics.functional import bert_score
+from torchmetrics.functional import bert_score as metrics_bert_score
 from torchmetrics.text import BERTScore
 from torchmetrics.utilities.imports import _BERTSCORE_AVAILABLE
 
+if _BERTSCORE_AVAILABLE:
+    from bert_score import score as original_bert_score
+
+os.environ["TOKENIZERS_PARALLELISM"] = "true"
```

Comment:
rather simple bool, string may case problem with cases
```suggestion
os.environ["TOKENIZERS_PARALLELISM"] = 1
```